src/core/Raycaster.js

```diff
@@ -82,8 +82,8 @@ Object.assign( Raycaster.prototype, {
 
 		} else if ( ( camera && camera.isOrthographicCamera ) ) {
 
-			this.ray.origin.set( coords.x, coords.y, ( camera.near + camera.far ) / ( camera.near - camera.far ) ).unproject( camera ); // set origin in plane of camera
-			this.ray.direction.set( 0, 0, - 1 ).transformDirection( camera.matrixWorld );
+			this.ray.origin.set( coords.x, coords.y, camera.near / ( camera.near - camera.far ) ).unproject( camera ); // set origin in plane of camera
+			this.ray.direction.set( 0, 0, 1 ).transformDirection( camera.matrixWorld );
```

Comment:
FYI: The default forward vector of a camera is `( 0, 0, - 1 )` and not `( 0, 0, 1 )` in `three.js`.